README.md

```diff
@@ -1,4 +1,6 @@
 PassFort
 ========
 
+[![CircleCI](https://circleci.com/gh/gocardless/passfort/tree/master.svg?style=svg)](https://circleci.com/gh/gocardless/passfort/tree/master)
+
 A Ruby client library for the [PassFort](https://passfort.com) API.
```

Comment:
I guess we can add installation / usage notes later on.

Reply:
Yeah, that's a good point. We'll push this to rubygems at some point, at which point the installation instructions will be `gem install passfort`